components/src/AppModalBundleConfiguration/appmodalbundleconfiguration.main.tsx

```diff
@@ -94,21 +94,25 @@ class AppModalBundleConfigurationMain extends React.Component<AppModalBundleConf
     };
     this.handleErrorMessage = this.handleErrorMessage.bind(this);
     this.handleQuantityChange = this.handleQuantityChange.bind(this);
+    this.handleConfiguratorAddToCart = this.handleConfiguratorAddToCart.bind(this);
+    this.handleRemove = this.handleRemove.bind(this);
   }
 
   componentDidMount() {
-    this.fetchDependantItemData();
+    const { bundleConfigurationItems } = this.props;
+    this.fetchDependantItemData(bundleConfigurationItems);
   }
 
-  componentWillReceiveProps(nextProps) {
+
+  componentDidUpdate(prevProps) {
     const { bundleConfigurationItems } = this.props;
-    if (bundleConfigurationItems !== nextProps.bundleConfigurationItems) {
-      this.fetchDependantItemData();
-    }
+    if (JSON.stringify(bundleConfigurationItems) !== JSON.stringify(prevProps.bundleConfigurationItems)) {
+        this.fetchDependantItemData(bundleConfigurationItems);
+      }
+    
   }
 
-  fetchDependantItemData() {
-    const { bundleConfigurationItems } = this.props;
+  fetchDependantItemData(bundleConfigurationItems) {
```

Comment:
Will this pass the linter when we re-add it? Just want to be sure. Here's the PR for the linter:
https://github.com/elasticpath/react-pwa-reference-storefront/pull/360

Reply:
Merged the linter branch #360 into this one and ran the linter inside `appModalBundleConfiguration` directory to fix all errors.  Since #360 is still undergoing changes should merge #360 before this one.

Reply:
Yep, I was wondering why it was like that in the first place. Cool, no further comments!